app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,45 @@
+.navbar {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+body {
+  background: image-url("raw/goofy-dad-bg.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+
+// HEADER STUFF
+.dad-header {
+  height: 50vh;
+  text-align: center;
+}
+
+.dad-header h1 {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.